flink-runtime/src/main/java/org/apache/flink/runtime/minicluster/MiniClusterConfiguration.java

```diff
@@ -54,12 +56,13 @@ public MiniClusterConfiguration(
 			Configuration configuration,
 			int numTaskManagers,
 			RpcServiceSharing rpcServiceSharing,
-			@Nullable String commonBindAddress) {
-
+			@Nullable String commonBindAddress,
+			boolean enableEmbeddedHaLeadershipControl) {
```

Comment:
I am always a bit more in favour of passing enum instead of booleans because enum values are more descriptive than `true` and `false`.